<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterDialog</class>
 <widget class="QDialog" name="PrinterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Small Icons/Icons/Prigram_Icon.png</normaloff>:/Small Icons/Icons/Prigram_Icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="printerNameLabel">
       <property name="text">
        <string>Printer Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="printerNameLineEdit">
       <property name="whatsThis">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="printerTypeLabel">
       <property name="text">
        <string>Printer Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="printerTypeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="prefix">
        <string>$ </string>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="depreciationTimeLabel">
       <property name="text">
        <string>Depreciation Time:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="depreciationTimeDoubleSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> hr</string>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="totalServiceCostLabel">
       <property name="text">
        <string>Total Service Cost:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <item>
        <widget class="QDoubleSpinBox" name="totalServiceCostDoubleSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="prefix">
          <string>$ </string>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="totalServiceCostCalculateButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Resource.qrc">
           <normaloff>:/Small Icons/Icons/Calculator_Icon.png</normaloff>:/Small Icons/Icons/Calculator_Icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="energeyConsumptionLabel">
       <property name="text">
        <string>Energy Consumption:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="energyConsumptionDoubleSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> kWh</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="depreciationLabel">
       <property name="text">
        <string>Depreciation:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="depreciationDoubleSpinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> $/hr</string>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="bedSizeLabel">
       <property name="text">
        <string>Bed Size:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="bedHeightLabel">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="bedLengthDoubleSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> mm</string>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="bedWidthDoubleSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> mm</string>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="bedHeightDoubleSpinBox">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> mm</string>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="bedLenghtLabel">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="bedWidthLabel">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="savePrinterButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>printerNameLineEdit</tabstop>
  <tabstop>printerTypeComboBox</tabstop>
  <tabstop>priceDoubleSpinBox</tabstop>
  <tabstop>depreciationTimeDoubleSpinBox</tabstop>
  <tabstop>totalServiceCostDoubleSpinBox</tabstop>
  <tabstop>totalServiceCostCalculateButton</tabstop>
  <tabstop>energyConsumptionDoubleSpinBox</tabstop>
  <tabstop>depreciationDoubleSpinBox</tabstop>
  <tabstop>bedLengthDoubleSpinBox</tabstop>
  <tabstop>bedWidthDoubleSpinBox</tabstop>
  <tabstop>bedHeightDoubleSpinBox</tabstop>
  <tabstop>savePrinterButton</tabstop>
 </tabstops>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>savePrinterButton</sender>
   <signal>clicked()</signal>
   <receiver>PrinterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
